ui/src/app/views/workspace-about.tsx

```diff
@@ -2,151 +2,276 @@ import {Component} from '@angular/core';
+            <TooltipTrigger content={WorkspaceInfoTooltipText()}>
+              <InfoIcon style={{margin: '0 0.3rem'}}/>
+            </TooltipTrigger>
+          </h3>
+          <div style={styles.infoBox} data-test-id='cdrVersion'>
+            <div style={styles.infoBoxHeader}>Dataset</div>
+            <div style={{fontSize: '0.5rem'}}>{cdrVersion ? cdrVersion.name : 'Loading...'}</div>
+          </div>
+          <div style={styles.infoBox} data-test-id='creationDate'>
+            <div style={styles.infoBoxHeader}>Creation Date</div>
+            <div style={{fontSize: '0.5rem'}}>{this.workspaceCreationTime}</div>
+          </div>
+          <div style={styles.infoBox} data-test-id='lastUpdated'>
+            <div style={styles.infoBoxHeader}>Last Updated</div>
+            <div style={{fontSize: '0.5rem'}}>{this.workspaceLastModifiedTime}</div>
+          </div>
+          <div style={styles.infoBox} data-test-id='dataAccessLevel'>
+            <div style={styles.infoBoxHeader}>Data Access Level</div>
+            <div style={{fontSize: '0.5rem'}}>{workspace ?
+              fp.capitalize(workspace.dataAccessLevel.toString()) : 'Loading...'}</div>
+          </div>
+          <Link disabled={!workspace}
+                onClick={() => this.setState({googleBucketModalOpen: true})}>Google Bucket</Link>
+          {!!this.workspaceClusterBillingProjectId() &&
+            <ResetClusterButton billingProjectId={this.workspaceClusterBillingProjectId()}/>}
         </div>
-        }
-        {profile.authorities.includes(Authority.FEATUREDWORKSPACEADMIN) &&
-          <div style={{display: 'flex', justifyContent: 'flex-end'}}>
-              <Button disabled={publishing} type='secondary'
-                onClick={() => this.publishUnpublishWorkspace(false)}>Unpublish</Button>
-              <Button onClick={() => this.publishUnpublishWorkspace(true)}
-                disabled={publishing} style={{marginLeft: '0.5rem'}}>Publish</Button>
-          </div>}
-      </FadeBox>;
-    }
+      </div>
+      {googleBucketModalOpen && <Modal>
+        <ModalTitle>Note</ModalTitle>
+        <ModalBody>
+            It is All of Us data use policy that researchers should not make copies of or download
+            individual-level data (including taking screenshots or other means of viewing
+            individual-level data) outside of the <i>All of Us</i> research environment without
+            approval from All of Us Resource Access Board (RAB).<br/>
+            Notebooks should rarely be downloaded directly from Google Cloud Console, as output
+            cells in Notebooks may contain sensitive individual level data.
+        </ModalBody>
+        <ModalFooter>
+          <Button type='secondary'
+                  onClick={() => this.setState({googleBucketModalOpen: false})}>Cancel</Button>
+          <Button onClick={() => this.openGoogleBucket()}
+                  style={{marginLeft: '0.5rem'}}>Continue</Button>
```

Comment:
It would be good if we could make this an actual <a> with an href for accessibility reasons (especially since its just opening a link) Is that doable?

Reply:
I'm not sure... I mean i don't see how to do that and have it also be a `Button` at the same time

Reply:
I think it's possible to make it a link, and simply style as a Button? Alternatively, wrap the continue within the button with an anchor link.